Components/Input/input.css

```diff
@@ -0,0 +1,30 @@
+.input-container{
+    max-width: 600px;
```

Comment:
use responsive units or rem unit in order to make it responsive. Otherwise overall is good.

Reply:
okay, will keep that in mind and do the changes.
Thanks for the feedback!!